### ISSUE TITLE
Typos: Self intro breakdown does not match original

### ISSUE THREAD

**Issue by JosueLugaro** [yangshun/tech-interview-handbook#630]
TLDR: Two small typos on the `/self-introduction/` page. Under "Example 1: ..." 30,000 should be changed to 40,000, and under "Example 2: ..." 4 should be changed to 3.

On this page `https://www.techinterviewhandbook.org/self-introduction/`:

Under "Example 1: Front End Engineer at Meta", the quote above "Mention something about the project which stands out.". The text reads

` "It receives over a million pageviews a month and is used by over 30,000 NUS undergraduates and even some professors."`

But in the "Self Introduction" it says:

` "It receives over a million pageviews a month and is used by over 40,000 NUS students and even some professors."`

I also something similar under "Example 2: Front End Engineer at Lyft", the text above "Acknowledge that you work with others. ..." reads:

`"I built the project with another engineer over the period of 4 months on a React/Redux and Golang stack."`

But the "Self Introduction" says:

`"I built the project with another engineer over the period of 3 months on a React/Redux and Golang stack."`

**Comment by Zume-z** [comment]
I have made a PR with your recommended typo updates, #632.

**Comment by yangshun** [comment]
Fixed on `main` by #632 (b60673b).
